auditbeat/auditbeat.reference.yml

```diff
@@ -86,12 +86,22 @@ auditbeat.modules:
 # sub-dictionary. Default is false.
 #fields_under_root: false
 
-# Internal queue size for single events in processing pipeline
-#queue_size: 1000
-
-# The internal queue size for bulk events in the processing pipeline.
-# Do not modify this value.
-#bulk_queue_size: 0
+# Internal queue configuration for buffering events to be published.
+#queue:
+  # Queue type by name (default 'mem')
+  #mem:
+    # Max number of events to store in the queue.
+    #events: 4096
```

Comment:
Semantics is a little different from old spooler here. The queue/broker does not flush its buffers. By default it basically stream the events right through to the outputs (if output takes somewhat longer it will be presented with all events available - up to bulk_max_size).

By configuring min_events, the queue will block the output until:
- at least min events is reached
- timeout is trigger

The `min events` is more of a hint, as the output might still receive a smaller buffer.